### ISSUE TITLE
The package is missing several files to run

### ISSUE THREAD

**Issue by Agetian** [Delphi79/Vas-Ort-Flam-RunUO-2.0#1]
Judging by the description and the fact that there are saves present, this package was meant to actually run. However, there are several important script files missing from the package (among them BaseCreature.cs and several AI-related files including BaseAI.cs, possibly others too, especially the ones under Scripts/Engines/AI), which makes it impossible to compile the package. Using stock RunUO 2.0 versions of those files doesn't seem to work either since they were most likely customized. There is a binary Scripts.CS.dll present, which hints at the fact that maybe it's possible to at least run the precompiled set of scripts, however, this file is ignored by the server package and is immediately recompiled when RunUO.exe is started (possibly because the .NET Framework version is different?), therefore, there is no apparent way to use the compiled binary either.

**Comment by Delphi79** [comment]
Yes this server package was given to us by one of the developers years ago. When we first got it in this condition we were also not able to compile it. I have reuploaded my fixed version that should allow you now to compile it. Complete with saves. Owner account is admin/password.

**Comment by Agetian** [comment]
Thank you very much for looking into this!

**Comment by Agetian** [comment]
By the way, if you don't mind me asking, do you know what client version should be used with this package? I tried a few, but I couldn't connect with either of them (5.0.9.1, some modern ones, tried 3.0.8 as well) - it'll accept the name and password but it'll drop the connection immediately and the game will respond with "There's trouble connecting to Origin".

**Comment by Delphi79** [comment]
There shouldn't really be a "specific" client version it accepts the way it is set up. I was able to connect using 5.0.9.1 when I tested it. The only way I could reproduce that error is by turning on OSI encryption in razor or by not using port 2597. I was able to connect using razor and ClassicUO. Make sure you are connecting to 127.0.0.1 on port 2597 if you are running it locally and that encryption is not on.
